File: src/sdk-reference/csharp/1/realtime/publish/snippets/publish.cs
try {
  string message = @"{ ""realtime"": ""rule the web"" }";

  kuzzle.realtime.publish("i-dont-exist", "in-database", message);

  Console.WriteLine("Message successfully published");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}

File: src/sdk-reference/cpp/1/realtime/publish/snippets/publish.cpp
try {
  std::string message = R"({ "realtime": "rule the web" })";

  kuzzle->realtime->publish("i-dont-exist", "in-database", message);

  std::cout << "Message successfully published" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
